### ISSUE TITLE
Wrong icons in file menu

### ISSUE THREAD

**Issue by PlaidDroid** [ChurchApps/FreeShow#613]
The import and export icons needs to be swapped.

![image](https://github.com/ChurchApps/FreeShow/assets/26256974/fb646d71-db7f-4ac3-ba42-0c42d71dc031)

**Comment by vassbo** [comment]
Good catch. I was thinking export=download and import=upload, but I see it probably makes more sense to swap them
